gtk/tree_view.go

```diff
@@ -333,17 +333,26 @@ func (v *TreeView) GetSearchEntry() *Entry {
 	return wrapEntry(glib.Take(unsafe.Pointer(c)))
 }
 
-// SetSearchEntry() is a wrapper around gtk_tree_view_set_search_entry().
+// SetSearchEntry is a wrapper around gtk_tree_view_set_search_entry().
 func (v *TreeView) SetSearchEntry(e *Entry) {
 	C.gtk_tree_view_set_search_entry(v.native(), e.native())
 }
 
+// SetSearchByStringMatch sets TreeView to search by string match.
+func (v *TreeView) SetSearchByStringMatch() {
```

Comment:
`StringMatch` seems confusing to me. `match` is usually used in the context of having a pattern and trying to match something against a pattern - which is not the case here.

Reply:
What about `SetSearchBySubstringMatch`? Because we are matching the model row against a search key as a substring of the row value.

Other ideas: `SetSearchSubstringMatchEqualFunc` because the C counterpart would be: `gtk_tree_view_set_search_equal_func(substring_match_equal_func)`. Or `SetSearchEqualSubstringMatch`.

Reply:
I disliked this name `SetSearchByStringMatch` as well, tks for the tips